sdcard/LOOKINGGLASS/PRESETS.TXT

```diff
@@ -9,10 +9,11 @@
 140,380,VHF MICS AND MARINE
 390,420,RADIOSONDES
 420,660,UHF MICS
+660,900, Water meters
```

Comment:
Remove space between ',' and description. Put it upper case.

Reply:
Also, are you sure the frequency range should start at 660 MHz?  Perhaps the 660 was copied from the line above, and it should be ~860?

I will also bring up the R900 water meter which operates in the ISM band 902-928MHz.  I'd suggest perhaps adding the ISM band to PRESETS.TXT, e.g. "902,928,ISM 900MHz" to pick up ISM devices.